appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/backstack/BackStackModel.kt

```diff
@@ -19,7 +19,7 @@ class BackStackModel<InteractionTarget : Any>(
 //    backPressHandler = backPressHandler,
 //    screenResolver = screenResolver,
 //    operationStrategy = operationStrategy,
-//    savedStateMap = savedStateMap,
+    savedStateMap = savedStateMap,
```

Comment:
let's add save/restore 2 tests for core models: backstack and spotlight

Reply:
done